scipy/stats/_multivariate.py

```diff
@@ -2620,3 +2621,295 @@ def entropy(self):
     method_frozen.__doc__ = doccer.docformat(
         method.__doc__, wishart_docdict_noparams)
     method.__doc__ = doccer.docformat(method.__doc__, wishart_docdict_params)
+
+
+_doc_multinom_callparam = """\
+n : int
+    Number of trials
+p : array_like, shape (k,)
+    Probabilities. These should sum to one. If they do not, then
+    ``p[-1]`` is modified to account for the remaining probability so that
+    ``sum(p) == 1``.
+"""
```

Comment:
Ouch. No, we're not. No longer we are. I'm going to quote this every time the topic of "why bothering with code reviews" is raises. Thanks!

Also rephrases the text a bit, is this better now?